src/Module/Admin/Logs/Settings.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Friendica\Module\Admin\Logs;
+
+use Friendica\Core\Config;
+use Friendica\Core\L10n;
+use Friendica\Core\Renderer;
+use Friendica\Core\System;
+use Friendica\Module\BaseAdminModule;
+use Friendica\Util\Strings;
+use Psr\Log\LogLevel;
+
+class Settings extends BaseAdminModule
+{
+	public static function post()
+	{
+		parent::post();
+
+		if (!empty($_POST['page_logs'])) {
+			parent::checkFormSecurityTokenRedirectOnError('/admin/logs', 'admin_logs');
+
+			$logfile   = (!empty($_POST['logfile']) ? Strings::escapeTags(trim($_POST['logfile'])) : '');
+			$debugging = !empty($_POST['debugging']);
+			$loglevel  = defaults($_POST, 'loglevel', LogLevel::ERROR);
+
+			Config::set('system', 'logfile', $logfile);
+			Config::set('system', 'debugging', $debugging);
+			Config::set('system', 'loglevel', $loglevel);
+		}
+
+		info(L10n::t("Log settings updated."));
+		self::getApp()->internalRedirect('admin/logs');
+	}
+
+	public static function content()
+	{
+		parent::content();
+
+		$log_choices = [
+			LogLevel::ERROR   => 'Error',
+			LogLevel::WARNING => 'Warning',
+			LogLevel::NOTICE  => 'Notice',
+			LogLevel::INFO    => 'Info',
+			LogLevel::DEBUG   => 'Debug',
+		];
+
+		if (ini_get('log_errors')) {
+			$phplogenabled = L10n::t('PHP log currently enabled.');
+		} else {
+			$phplogenabled = L10n::t('PHP log currently disabled.');
+		}
+
+		$t = Renderer::getMarkupTemplate('admin/logs/settings.tpl');
+
+		return Renderer::replaceMacros($t, [
+			'$title' => L10n::t('Administration'),
+			'$page' => L10n::t('Logs'),
+			'$submit' => L10n::t('Save Settings'),
+			'$clear' => L10n::t('Clear'),
+			'$baseurl' => System::baseUrl(true),
+			'$logname' => Config::get('system', 'logfile'),
+			// name, label, value, help string, extra data...

```

Comment:
I think this comment can be removed, as the structure of the fields is well documented in the docs.